modular_citadel/code/modules/custom_loadout/custom_items.dm

```diff
@@ -397,3 +397,11 @@
 	desc = "Comfy Lucky Jackboots with the word Luck on them."
 	item_state = "luckyjack"
 	icon_state = "luckyjack"
+
+/obj/item/clothing/under/lunasune
+	name = "Divine Robes"
+	icon = 'icons/obj/custom.dmi'
+	icon_override = 'icons/mob/custom_w.dmi'
```

Comment:
should probably change this to `alternate_worn_icon`

Reply:
I don't know how to do either of those things, it took me the help of 3 contributors to do it all.

Reply:
Fixed CL

Reply:
after exploring, I figured both things out, id like to leave it as is